### PR TITLE
Support matmul+bias+gelu fusion on GPUs

### PR DESCRIPTION
**Pull request by kaixih**
This PR enables the fusion pattern of MatMul+BiasAdd+Gelu on GPUs when CublasLt is enabled by `TF_USE_CUBLASLT=1`.

cc. @nluehr

### CODE REVIEW

**Review by ezhulenev — APPROVED** [review]

**Review by ezhulenev — APPROVED** [review]

**Comment by kaixih** [comment]
Force pushed the PR to resolve the conflict.

**Comment by njzjz** [comment]
Hi kaixih, does cublaslt support float64? Thank you.

**Comment by kaixih** [comment]
> Hi kaixih, does cublaslt support float64? Thank you.

Hi @njzjz As for this PR, the cublaslt doesn't support float64 for fusion. But cublaslt itself support fp64.

**Comment by kaixih** [comment]
Any update/feedback for this PR that I can work on?

**Comment by kaixih** [comment]
Hi, @ezhulenev , can you take a look at this PR? It seems something is blocking the merge. Thanks.

**Review by reedwm — APPROVED** [review]

**Comment by kaixih** [comment]
@reedwm Can you take a look at if the unhandled switch case is gone?

**Comment by kaixih** [comment]
Gentle ping @reedwm .

**Review by reedwm — APPROVED** [review]

**Comment by kaixih** [comment]
Rebased to resolve the conflicts. PTAL. @reedwm

**Comment by kaixih** [comment]
I can see some tests are failed but the log shows the failure is unrelated to this PR. Can you help check? @reedwm

**Comment by gbaned** [comment]
Hi @kaixih Can you please resolve conflicts? Thank you!

**Comment by kaixih** [comment]
Rebased to resolve conflicts. PTAL. @reedwm

**Review by reedwm — APPROVED** [review]

File: tensorflow/stream_executor/cuda/cuda_blas_lt.cc [hunk 1]
```diff
@@ -63,6 +64,13 @@ cublasLtEpilogue_t AsCublasLtEpilogue(BlasLt::Epilogue epilogue) {
       return CUBLASLT_EPILOGUE_BIAS;
     case BlasLt::Epilogue::kBiasThenReLU:
       return CUBLASLT_EPILOGUE_RELU_BIAS;
+    case BlasLt::Epilogue::kBiasThenGeLUApproximate:
```
Comment (SandSnip3r) [inline tensorflow/stream_executor/cuda/cuda_blas_lt.cc:67]: enumeration value 'kGeLU' not handled in switch

Comment (kaixih) [reply tensorflow/stream_executor/cuda/cuda_blas_lt.cc:74]: Resolved. PTAL.

**Review by SandSnip3r — APPROVED** [review]

**Pull request merged**